monitors/filterWords.js

```diff
@@ -1,42 +1,44 @@
-exports.conf = {
-  enabled: true,
-  // Change to false if the filter should not ignore other bots
-  ignoreBots: true,
-  // Change to true if the filter should ignore selfBots
-  ignoreSelf: false,
-};
-
-exports.run = (client, msg) => {
-  // Please add the channel ID you want to recieve these reports in.
-  const modlog = "CHANNELIDHERE";
+exports.run = async (client, msg) => {
+  const modlog = msg.guild.settings.modLog;
   if (msg.deletable) {
-    const badWords = msg.guild.settings.swearWords.filter(word => msg.content.toLowerCase().includes(word));
-    if (badWords.length > 0) {
-      let badWordList = [];
-      for (let i = 0; i < badWords.length; i++) {
-        badWordList += badWords[i];
+    if (msg.guild.settings.swearWords) {
```

Comment:
I believe this an array, correct? If that's the case, you should check to see if the array as something in it (aka a word), in addition to checking if the setting exists.